File: cSharp/Bebop.Handlers/StrikeMsg.cs
using Bebop.Attributes;
using Bebop.Runtime;
using PacTheMan.Models;
using System;

namespace lean_pactheman_client {

    [RecordHandler]
    public static class SessioStrikeMsgHandler {

        [BindRecord(typeof(BebopRecord<StrikeMsg>))]
        public static void HandleSessionMsg(object client, StrikeMsg msg) {
            GameState.Instance.SignalStrike();
            Console.Write($"Watch out this is your {GameState.Instance.StrikeCount} strike. ");
            Console.WriteLine($"Only {Constants.MAX_STRIKE_COUNT - GameState.Instance.StrikeCount} strike(s) left.");
        }
    }
}

File: typescript/src/bebop_handlers/strikeMsg.ts
import { Constants } from '..';
import GameState from '../classes/gameState';
import { StrikeMsg } from '../models/pactheman.models';
import BebopHandler from './bebopHandler';

export default class StrikeMsgHandler implements BebopHandler {
    handleMessage(sender: any, message: any): void {
        message = StrikeMsg.decode(message);
        GameState.Instance.strikeCount++;
        console.log(`Watch out this is your ${GameState.Instance.strikeCount} strike.`);
        console.log(`Only ${Constants.MAX_STRIKE_COUNT - GameState.Instance.strikeCount} strike(s) left.`);
    }
}